src/wp-includes/comment-template.php

```diff
@@ -1275,14 +1275,14 @@ function wp_comment_form_unfiltered_html_nonce() {
  * Will not try to get the comments if the post has none.
  *
  * @since WP-1.5.0
+ * @since WP-5.5.0 Removed the need to use the $user_ID global.
  *
```

Comment:
This was removed in https://core.trac.wordpress.org/changeset/48140:

```suggestion
 * @since WP-1.5.0
 *
```